.github/workflows/infra_drift_detection.yml

```diff
@@ -0,0 +1,184 @@
+name: Drift Detection
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: Environment where the resources will be deployed.
+        type: string
+        required: true
+        default: dev
+      base_path:
+        description: The base path on which the script will look for Terraform projects
+        type: string
+        required: true
+        default: infra/resources/
+      env_vars:
+        description: List of environment variables to set up, given in env=value format.
+        type: string
+        required: false
+      use_private_agent:
+        description: Use a private agent to run the Terraform plan.
+        type: boolean
+        required: false
+        default: false
+      webhook_url:
+        description: The webhook URL to send the notification to.
+        type: string
+        required: false
+  schedule:
+    - cron: '00 08 * * *'
+
+
+env:
+  ARM_SUBSCRIPTION_ID: ${{ secrets.ARM_SUBSCRIPTION_ID }}
+  ARM_TENANT_ID: ${{ secrets.ARM_TENANT_ID }}
+  ARM_USE_OIDC: true
+  ARM_USE_AZUREAD: true
+  ARM_STORAGE_USE_AZUREAD: true
+  BASE_DIR: ${{ inputs.base_path != '' && inputs.base_path || 'infra/resources/' }}
+  AZURE_ENVIRONMENT: ${{ inputs.environment != '' && inputs.environment || 'dev' }}
```

Comment:
As we don't have a `dev` environment, I would stick to use `prod` as default. What do you think?

Reply:
I don't think there are any problems, I used `dev` just to use a test environment. But now I change the various references to `prod`